pkg/cmd/server/command_test.go

```diff
@@ -164,11 +164,26 @@ func TestCommandBindingHostname(t *testing.T) {
 	}
 }
 
-func TestCommandBindingNodes(t *testing.T) {
+// AllInOne always adds the default hostname
+func TestCommandBindingNodesForAllInOneAppend(t *testing.T) {
 	valueToSet := "first,second,third"
 	actualCfg := executeCommand([]string{"--nodes=" + valueToSet})
 
 	expectedConfig := NewDefaultConfig()
+	expectedConfig.NodeList.Set(expectedConfig.Hostname + "," + valueToSet)
```

Comment:
toLower hostname in case the detected hostname is mixed-case? also, doesn't this depend on alpha ordering?